.vsts.template.nonwindows.yml

```diff
@@ -20,9 +20,14 @@ steps:
   displayName: Build & Layout Release
 
 # Run test
-- script: ./dev.sh test
+- script: ./dev.sh testl0
   workingDirectory: src
-  displayName: Test
+  displayName: Unit tests
+
+- script: ./dev.sh testl1
+  workingDirectory: src
+  displayName: Functional tests
```

Comment:
Nit: Should we call this L1 tests to keep everything consistent?

Reply:
I intentionally stayed away from that because it doesn't really mean anything to non-Microsoft people. Didn't want to change all references to L0, but this at least provides a good idea of what it is to anyone who fails CI because of them.